src/OnboardingSPA/components/Drawer/DrawerPanel/DesignColors.js

```diff
@@ -526,10 +635,10 @@ const DesignColors = () => {
 				{ showColorPicker && (
 					<Popover>
 						<div
-							className="custom-palette__picker-close-icon"
-							onClick={ () => setShowColorPicker( false ) }
 							role="button"
 							tabIndex={ 0 }
+							className="custom-palette__picker-close-icon"
+							onClick={ () => setShowColorPicker( false ) }
```

Comment:
What's `#874141` in line 648?

Reply:
That is the default value for the color picker when opened, will remove that.